application/models/User.php

```diff
@@ -970,17 +965,40 @@ public function getGroupMemberListButtons()
     }
 
     /**
-     * Returns true if logged in user with id $loginId can edit this user
+     * Return true if user with id $managerId can edit this user
+     * @param int|null $managerId default to current user
      *
-     * @param int $loginId
      * @return bool
      */
-    public function canEdit($loginId)
+    public function canEdit($managerId = null)
```

Comment:
So, this method is more like a `caneBeEditedBy` ?
Maybe we can rename this? (I guess is not widely used)

Reply:
It's the same name than before, renaming it broke API version …

I add again $managerId because you say it broke API versin (and you're right) … then  : **no, i don't rename it**  it's not a new function.

Reply:
> So, this method is more like a `caneBeEditedBy` ? Maybe we can rename this? (I guess is not widely used)

https://github.com/LimeSurvey/LimeSurvey/pull/3495#discussion_r1338537273 please …